.travis.yml

```diff
@@ -22,7 +22,8 @@ env:
   - MAVEN_JAVA_VERSION=3-jdk-11-slim INFLUXDB_VERSION=1.3
   - MAVEN_JAVA_VERSION=3-jdk-11-slim INFLUXDB_VERSION=1.2
   - MAVEN_JAVA_VERSION=3-jdk-11-slim INFLUXDB_VERSION=1.1
-  - MAVEN_JAVA_VERSION=3-jdk-8-slim INFLUXDB_VERSION=1.7 SNAPSHOT=Y
+    # switched to alpine due to "slim" issue https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=911925
+  - MAVEN_JAVA_VERSION=3-jdk-8-alpine INFLUXDB_VERSION=1.7 SNAPSHOT=Y
```

Comment:
This change is not needed anymore, the bug was fixed with a newer jacoco plugin.

Reply:
This change was reverted.